developers/weaviate/current/configuration/authentication.md

```diff
@@ -125,7 +113,8 @@ services:
       # the groups. Groups must be an array of string. If groups_claim is not set
       # weaviate will not try to extract groups and pass an empty array to the 
       # authorization module.      
-      AUTHENTICATION_OIDC_GROUPS_CLAIM: 'groups'
+      # NOTE: Groups are not yet implemented in Weaviate's authorization module. Please do not use groups option for now.
+      # AUTHENTICATION_OIDC_GROUPS_CLAIM: 'groups'
```

Comment:
We still mention groups as something thats working above, maybe we should also remove or comment that out?

Reply:
Do you mean in here? `from which different [authorization](./authorization.html) options can be specified for different groups of users`?

What if I rephrase "groups of users" to "sets of users", as those "groups" are not OIDC groups anyway?

Reply:
I meant in general, we refered groups at a few places and I think we should leave those out, until it actually works

Reply:
Okay. I have removed all references to `group`, and the one above has been changed to `subset of admin users` as per the above discussion.